tsdb/engine/tsm1/file_store_test.go

```diff
@@ -2253,9 +2253,9 @@ func TestFileStore_Stats(t *testing.T) {
 		"mem": []tsm1.Value{tsm1.NewValue(0, 1.0)},
 	})
 
-	replacement := files[2] + "-foo" + ".tmp" // Assumes new files have a .tmp extension
+	replacement := fmt.Sprintf("%s.%s.%s", files[2], tsm1.TmpTSMFileExtension, tsm1.TSMFileExtension) // Assumes new files have a .tmp extension
```

Comment:
@benbjohnson I changed the implementation to check for either `.tsm.tmp` or `.tsm`. The previous test was creating a filename ending in `-foo.tmp`, which didn't match the new filters, so I updated the test to come up with a new filename that matched one of the filters.